src/utils/fileSystem.ts

```diff
@@ -0,0 +1,11 @@
+import fs from 'fs';
+import { HermesCPUProfile } from '../types/hermesProfileInterfaces';
+
+export const readData = (path: string): HermesCPUProfile => {
```

Comment:
Defines a readData function, only added for simplicity as JSON Parse, readFileSync and others could fail and this basically encapsulates them in a single block